src/components/Article.tsx

```diff
@@ -73,7 +73,7 @@ function Article() {
 
       <form className="card comment-form">
         <div className="card-block">
-          <textarea className="form-control" placeholder="Write a comment..." rows="3"></textarea>
+          <textarea className="form-control" placeholder="Write a comment..."  /* rows="3" */></textarea>
```

Comment:
Will need to find a way to append some node_modules type definitions from outside the package that is using them - or find a different way to solve it :thinking: